composer.json

```diff
@@ -1,23 +1,45 @@
 {
-  "name": "",
-  "description": "",
-  "require-dev": {
-    "phpunit/phpunit": "^9.1",
-    "friendsofphp/php-cs-fixer": "^2.0"
-  },
-  "require": {},
-  "scripts": {
-    "test": "phpunit",
-    "php-cs-fixer": "php-cs-fixer fix ./"
-  },
-  "autoload": {
-    "psr-4": {
-      "App\\": "src"
+    "name": "Phenom-Backend",
```

Comment:
I got the error: 

> Deprecation warning: Your package name phenom-backend is invalid, it should have a vendor name, a forward slash, and a package name ...  Make sure you fix this as Composer 2.0 will error.

Changing it to "_phenom/backend_" (all lower case) fixes that.